### PR TITLE
Add Behat testing with Firefox

### PR DESCRIPTION
**Pull request by mvorisek**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -65,7 +65,7 @@ script:
     #- mkdir -p coverage
     #- chmod 777 coverage
     #- cp tools/coverage.php demos/coverage.php
-    - vendor/bin/behat --config behat-travis.yml || exit -1
+    - vendor/bin/behat --config tests/behat-travis.yml || exit -1
```
Comment (georgehristov) [inline .travis.yml]: tests/behat/travis.yml?

Comment (mvorisek) [reply .travis.yml]: I do not like too much files in root directory, but I think it should not go to tests/behat, it is config... Let's clear this on Discord.

File: .travis.yml [hunk 2]
```diff
@@ -65,7 +65,7 @@ script:
     #- mkdir -p coverage
     #- chmod 777 coverage
     #- cp tools/coverage.php demos/coverage.php
-    - vendor/bin/behat --config behat-travis.yml || exit -1
+    - vendor/bin/behat --config behat.yml.travis || exit -1
```
Comment (mvorisek) [inline .travis.yml:68]: so you can/should create `behat.yml.local`

**Review by georgehristov — APPROVED** [review]
👍

**Pull request merged**